cmd/corectl/main.go

```diff
@@ -298,6 +300,72 @@ func reset(client *rpc.Client, args []string) {
 	}
 }
 
+func grant(client *rpc.Client, args []string) {
+	editAuthz(client, args, "grant")
+}
+
+func revoke(client *rpc.Client, args []string) {
+	editAuthz(client, args, "revoke")
+}
+
+type grantReq struct {
+	Policy     string
+	Grant_Type string
+	Grant_Data interface{}
+}
+
+func editAuthz(client *rpc.Client, args []string, action string) {
+	usage := "usage: corectl " + action + " [-cn name] [-ou name] [-t token] [policy]"
+	var flags flag.FlagSet
+	flagT := flags.String("t", "", action+" `access-token` access")
+	flagCN := flags.String("cn", "", action+" X.509 `Common Name` access")
+	flagOU := flags.String("ou", "", action+" X.509 `Organizational Unit` access")
+
+	flags.Usage = func() {
+		fmt.Println(usage)
+		flags.PrintDefaults()
+		os.Exit(1)
+	}
+	flags.Parse(args)
+	args = flags.Args()
+	if len(args) < 2 {
```

Comment:
I may not be up to speed on how `flags` works, but it looks like we're only using the first item in `args`. This check implies that there is at least one more argument we care about?